WTWSMS/common/religions/00_religions.txt

```diff
@@ -4337,7 +4330,6 @@ pagan_group = {
 		max_consorts = 3
 		
 		ai_convert_same_group = 0 # never
-		ai_convert_other_group = 1 # only when zealous

```

Comment:
OK, I deleted 1 because Cybelians had 2. Could you change it? I'm afraid I won't be able to push any commits today.